examples/python/ocr/README.md

```diff
@@ -0,0 +1,156 @@
+The classification of layouts and the text detection (including confidence levels) are visualized in the Rerun viewer.
+Finally, the recovery text document section presents the restored document with sorted order. By clicking on the restored text, the text area will be highlighted.
+
+## Logging and visualizing with Rerun
+The visualizations in this example were created with the following Rerun code.
+
+### Image
+The input document is logged as [`Image`](https://www.rerun.io/docs/reference/types/archetypes/image) object to the `Image` entity.
+```python
+rr.log("Image", rr.Image(coloured_image))
+```
+
+### Label mapping
+
+An annotation context is logged with a class ID and a color assigned per layout type using [`AnnotationContext`](https://rerun.io/docs/reference/types/archetypes/annotation_context).
+
+```python
+class LayoutType(Enum):
+    UNKNOWN = (0, "unknown", Color.Purple)
+    TITLE = (1, "title", Color.Red)
+    TEXT = (2, "text", Color.Green)
+    FIGURE = (3, "figure", Color.Blue)
+    FIGURE_CAPTION = (4, "figure_caption", Color.Yellow)
+    TABLE = (5, "table", Color.Cyan)
+    TABLE_CAPTION = (6, "table_caption", Color.Magenta)
+    REFERENCE = (7, "reference", Color.Purple)
+    FOOTER = (7, "footer", Color.Orange)
+
+    @property
+    def number(self):
+        return self.value[0]  # Returns the numerical identifier
+
+    @property
+    def type(self):
+        return self.value[1]  # Returns the type
+
+    @property
+    def color(self):
+        return self.value[2]  # Returns the color
+
+    @classmethod
+    def get_annotation(cls):
+        return [(layout.number, layout.type, layout.color) for layout in cls]
+
+
+def detect_and_log_layout(img_path):
+    rr.log(
+        "Image",
+        # The annotation is defined in the Layout class based on its properties
+        rr.AnnotationContext(LayoutType.get_annotation()),
+        timeless=True
+    )
+```
+
+### Detections
+The detections include the layout types and the text detections. Both of them are logged as [`Boxes2D`](https://www.rerun.io/docs/reference/types/archetypes/boxes2d) to Rerun.
+
+```python
+rr.log(
+    base_path,
+    rr.Boxes2D(
+        array=record['bounding_box'],
+        array_format=rr.Box2DFormat.XYXY,
+        labels=[str(layout_type.type)],
+        class_ids=[str(layout_type.number)]
+    ),
+    rr.AnyValues(name=record_name),
+    timeless=True
+)
+```
+In the detection of the text, the detection id and the confidence are given supplementary.
+```python
+rr.log(
+    f"{base_path}/Detections/{detection['id']}",
+    rr.Boxes2D(
+        array=detection['box'],
+        array_format=rr.Box2DFormat.XYXY,
+        class_ids=[str(layout_type.number)]
+    ),
+    rr.AnyValues(
+        DetectionID=detection['id'],
+        Text=detection['text'],
+        Confidence=detection['confidence']
+    ),
+    timeless=True
+)
+```
+
+### Setting up the blueprint
+
+The blueprint for this example is created by the following code:
```

Comment:
a new user has no idea what a blueprint is and what this is going to do. Should write something about that this sets up the viewer layout etc.